plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -199,4 +199,7 @@ private MLCommonsSettings() {}
 
     public static final Setting<Boolean> ML_COMMONS_CONNECTOR_PRIVATE_IP_ENABLED = Setting
         .boolSetting("plugins.ml_commons.connector.private_ip_enabled", false, Setting.Property.NodeScope, Setting.Property.Dynamic);
+
+    public static final Setting<Boolean> ML_COMMONS_CONTROLLER_ENABLED = Setting
+        .boolSetting("plugins.ml_commons.controller_enabled", true, Setting.Property.NodeScope, Setting.Property.Dynamic);
```

Comment:
It is Controller and not Model Controller, so I would insist this name.

Reply:
Controller for what? Model or Connector or ML-Commons?

Reply:
Eventually we want to generalize it for whole ML plugin

Reply:
You can generalize the feature, but the name needs to be specific so that users know what controller it's for. Using a general name doesn't seem a good idea to me.

Reply:
@zane-neo do you have any suggestion? 

We already published this feature as controller: https://opensearch.org/docs/latest/ml-commons-plugin/api/controller-apis/index/